incubator/hnc/pkg/reconcilers/hnc_config.go

```diff
@@ -235,6 +241,25 @@ func (r *ConfigReconciler) syncActiveReconcilers(ctx context.Context, inst *api.
 	return nil
 }
 
+// isTypeCofigured keeps track of whether the configuration of a type already exists.
+func (r *ConfigReconciler) isTypeConfigured(inst *api.HNCConfiguration, t api.TypeSynchronizationSpec,
+	mp map[schema.GroupVersionKind]bool) bool {
+	gvk := schema.FromAPIVersionAndKind(t.APIVersion, t.Kind)
+	if mp[gvk] {
```

Comment:
Probably more clear to invert this:
```
if !mp[gvk] {
  mp[gvk] = true
  return false
}
... everything else ...
```

go/gocomments#indent-error-flow (Google-internal link) is not _exactly_ what we mean here since this isn't an "error" condition, but the fundamental guideline of "reduce levels of indentation" does apply here.

Reply:
Thanks for the pointer. Updated the code to follow the guidance.